OpenRA.Mods.Common/UpdateRules/UpdatePath.cs

```diff
@@ -53,7 +53,8 @@ public class UpdatePath
 				new RemovePaletteFromCurrentTileset(),
 				new DefineLocomotors(),
 				new DefineOwnerLostAction(),
-				new RenameEmitInfantryOnSell()
+				new RenameEmitInfantryOnSell(),
+				new SplitRepairDecoration(),
```

Comment:
I've deliberately added a comma here in the hopes that it will reduce the rebase conflicts going forward.

Reply:
Makes sense, I'll try to keep that in mind for my PRs as well.